packages/wallet-stamper/.env.example

```diff
@@ -0,0 +1,5 @@
+# Public API key, obtain from your API provider
```

Comment:
This env file is only needed for local testing correct?

Reply:
yup! Ill add comment